prng.c

```diff
@@ -1,22 +1,22 @@
 #include <unistd.h>
 #include <stdio.h>
+#include <stdint.h>
 #include <stdlib.h>
 #include <string.h>
 #include <x86intrin.h>
 #define BUFSIZE (1<<14)
-#define SEEDTYPE __uint64_t
+#define SEEDTYPE uint64_t
 #include "./prng.h"
-typedef struct args { __int64_t bytes; SEEDTYPE seed[4]; int rval; } args_t;
+typedef struct args { uint64_t bytes; SEEDTYPE seed[4]; int rval; } args_t;
 args_t parseArgs(int argc, char **argv);
-void prng_gen(prng_state *s, __uint64_t buf[], __uint64_t size);
 
 int main(int argc, char **argv) {
   args_t a = parseArgs(argc, argv);
   if (a.rval < 0) { return a.rval; }
   prng_state s = prng_init(a.seed);
-  __uint64_t buf[BUFSIZE] __attribute__ ((aligned (64)));
-  __int64_t cycles = 0, start;
-  for (__int64_t bytes = a.bytes; bytes >= 0; bytes -= sizeof(buf)) {
+  uint64_t buf[BUFSIZE] __attribute__ ((aligned (64)));
+  int64_t cycles = 0, start;
+  for (uint64_t bytes = a.bytes; bytes >= 0; bytes -= sizeof(buf)) {
```

Comment:
`bytes` need to be signed; the loop never terminates here.

Reply:
Oops.